src/pilgrim/ui/screens/modals/edit_photo_modal.py

```
@@ -12,15 +12,11 @@ def __init__(self, photo: Photo):
 
-    def _generate_photo_hash(self, photo: Photo) -> str:
-        """Generate a short, unique hash for a photo"""
-        unique_string = f"{photo.name}_{photo.id}_{photo.addition_date}"
-        hash_object = hashlib.md5(unique_string.encode())
-        return hash_object.hexdigest()[:8]
+
 
     def compose(self) -> ComposeResult:
         # Generate hash for this photo
-        photo_hash = self._generate_photo_hash(self.photo)
+        photo_hash = None
```

Ruff (0.12.2)

Local variable `photo_hash` is assigned to but never used

Remove assignment to unused variable `photo_hash`

(F841)